.github/workflows/load-workflow-variables-test.yml

```diff
@@ -29,22 +29,22 @@ jobs:
       - name: 'Load Env Vars'
         uses: './.github/actions/load-workflow-variables' # ratchet:exclude
         with:
-          working_directory: '.github/actions/load-workflow-variables/test/terraform/org'
-          filepath: '${{ github.workspace }}/.github/actions/load-workflow-variables/test/test.env.yml'
+          working_directory: 'terraform/org'
+          filepath: '.github/actions/load-workflow-variables/test.env.yml'
```

Comment:
Why did this filepath change?
```shell
OLD:  .github/actions/load-workflow-variables/test/test.env.yml
New:  .github/actions/load-workflow-variables/test.env.yml
```

Reply:
I see now, file was moved. GH UI I thought usually detected that as a move rather than an addition and deletion.